pkgs/native_assets_builder/test_data/manifest.yaml

```diff
@@ -93,6 +93,18 @@
 - simple_data_asset/bin/deep_modify_data_asset.dart.debug
 - some_dev_dep/bin/some_dev_dep.dart
 - some_dev_dep/pubspec.yaml
+- treeshaking_native_libs/pubspec.yaml
```

Comment:
This project is not exercised. Please add a test in  native_assets_builder/test/ that runs this project.

(It's only being `dart pub get` in the `native.yaml`.)